<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillingTest</class>
 <widget class="QDialog" name="BillingTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BillingState</string>
  </property>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Message to Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="billingButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Billing Request</string>
   </property>
  </widget>
  <widget class="QLabel" name="responseLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Response</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="stateText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current State</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="responseText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>221</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
